src/main/java/com/jpaweekily/domain/order/infrastructrue/OrderProductRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.jpaweekily.domain.order.infrastructrue;
+
+import com.jpaweekily.domain.order.OrderProduct;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface OrderProductRepository extends JpaRepository<OrderProduct, Long> {
+}
```

Comment:
```suggestion
public interface OrderProductRepository extends JpaRepository<OrderProduct, Long>, CustomOrderProductRepository {
}
```

이렇게 `CustomOrderProductRepository` 인터페이스를 새로 만들고 saveAll() 메서드를 정의한다음 구현체를 만들어주면, OrderService에서 OrderProductRepository에 의존하는 형태로 할 수 있을 것 같아요. 레포지토리 이름은 고민해보고 결정해주면 좋을 것 같습니다.

Reply:
c2c201c00bd733ba55743745440a9bbfc1eab803
적용하였습니다! 네임이 부분에서는 기존에 사용하던 `OrderProductBulkRepository`라는 네이밍을 사용할까 했지만, 너무 한정적인 클래스가 되어버리는 것 같아서 예시로 말씀해주신  `CustomOrderProductRepository` 를 사용하였습니다!